packages/vuetify/src/components/VVirtualScroll/VVirtualScroll.tsx

```diff
@@ -72,11 +72,19 @@ export const VVirtualScroll = genericComponent<new <T, Renderless extends boolea
 
     useToggleScope(() => props.renderless, () => {
       onMounted(() => {
-        containerRef.value = getScrollParent(vm.vnode.el as HTMLElement, true)
-        containerRef.value?.addEventListener('scroll', handleScroll)
+        containerRef.value = getScrollParent(vm.vnode.el as HTMLElement, false)
+        if (containerRef.value === document.documentElement) {
```

Comment:
I tested in multiple live html pages:

- `document.documentElement.addEventListener("scroll", (event) => {console.log("htmltag scrolling");});` doesn't work

- However, `document.addEventListener("scroll", (event) => {console.log("document scrolling");});` works

Reply:
Yeah idk some archaic netscape navigator shit probably. document.documentElement.scrollTop is the one that changes.

Reply:
What's the status on this feedback?